collector/reloads_config.go

```diff
@@ -0,0 +1,21 @@
+package collector
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type reloadsConfigCollector struct {
+	Failures  *prometheus.Desc
+	Successes *prometheus.Desc
+}
+
+func newReloadsConfigCollector() *reloadsConfigCollector {
+	desc := newDescFunc(namespace, "reloads_config")
+	return &reloadsConfigCollector{
+		Failures:  desc("failures", "Number of failures during config reload."),
```

Comment:
I think the type of successes and failures are counter.
How about changing it to failures_total and successes_total?